### ISSUE TITLE
Namespace issues for Airflow example with package version 0.16.0

### ISSUE THREAD

**Issue by MichoelSnow** [MarquezProject/marquez#1504]
The airflow example does not correctly populate the marquez database when using `marquez-airflow==0.16.0`.  These issues do not occur when setting `marquez-airflow==0.15.2` in the requirements.txt file. 

Below are SQL queries against the postgres db after following the instructions in the airflow example docs and using `marquez-airflow==0.16.0`.

```sql
marquez=# select name from namespaces;
           name
--------------------------
 default
 example
 postgres://postgres:5432
(3 rows)
```

```sql
marquez=# select name, namespace_name, source_name from datasets;
         name          |      namespace_name      |       source_name
-----------------------+--------------------------+--------------------------
 example.public.sums   | postgres://postgres:5432 | postgres://postgres:5432
 example.public.counts | postgres://postgres:5432 | postgres://postgres:5432
(2 rows)
```
This causes issues in the Marquez UI when it tries to call the data from that namespace (as seen below)

![image](https://user-images.githubusercontent.com/28357266/125462386-75631d61-4899-47db-b29b-8188d9a2e861.png)

docker --version
Docker version 19.03.0, build aeac949

docker-compose --version
docker-compose version 1.24.1, build 4667896

System specs: 

```
System:    Host: ip-10-100-27-209 Kernel: 4.4.0-1121-aws x86_64 (64 bit gcc: 5.4.0) Console: tty 8
           Distro: Ubuntu 16.04 xenial
Machine:   System: Amazon EC2 product: m5.4xlarge
           Mobo: Amazon EC2 model: N/A Bios: Amazon EC2 v: 1.0 date: 10/16/2017
CPU:       Octa core Intel Xeon Platinum 8175M (-HT-MCP-) cache: 33792 KB
           flags: (lm nx sse sse2 sse3 sse4_1 sse4_2 ssse3) bmips: 40000
           clock speeds: max: 2500 MHz 1: 2500 MHz 2: 2500 MHz 3: 2500 MHz 4: 2500 MHz 5: 2500 MHz 6: 2500 MHz
           7: 2500 MHz 8: 2500 MHz 9: 2500 MHz 10: 2500 MHz 11: 2500 MHz 12: 2500 MHz 13: 2500 MHz 14: 2500 MHz
           15: 2500 MHz 16: 2500 MHz
Graphics:  Card: Amazon.com Device 1111 bus-ID: 00:03.0
           Display Server: N/A driver: N/A tty size: 120x30 Advanced Data: N/A out of X
Network:   Card: Amazon.com Elastic Network Adapter (ENA) driver: ena v: 2.1.0K bus-ID: 00:05.0
           IF: eth0 state: up speed: N/A duplex: N/A mac: <filter>
Drives:    HDD Total Size: NA (-) ID-1: /dev/nvme1n1 model: N/A size: 274.9GB
           ID-2: /dev/nvme0n1 model: N/A size: 80.5GB
Partition: ID-1: / size: 73G used: 71G (97%) fs: ext4 dev: /dev/nvme0n1p1
RAID:      No RAID devices: /proc/mdstat, md_mod kernel module present
Sensors:   None detected - is lm-sensors installed and configured?
Info:      Processes: 445 Uptime: 153 days Memory: 13790.9/62958.0MB Init: systemd runlevel: 5 Gcc sys: 5.4.0
           Client: Shell (bash 4.3.481) inxi: 2.2.35
```

**Comment by mobuchowski** [comment]
I think this PR already fixes this issue: https://github.com/MarquezProject/marquez/pull/1502

**Comment by MichoelSnow** [comment]
Setting `marquez-airflow==0.16.1rc1` in the requriements.txt does not fix the issue.

**Comment by phixMe** [comment]
The official release will be coming very soon with the fix for this one. Thanks so much for reporting this issue with a great write-up.

**Comment by MichoelSnow** [comment]
Thanks for the heads up (and the upcoming fix).

**Comment by wslulciuc** [comment]
@MichoelSnow, closing as this was fixed in `0.16.1` and [you were able to confirm](https://marquezproject.slack.com/archives/C01E8MQGJP7/p1626314345120800?thread_ts=1625494255.098000&cid=C01E8MQGJP7).
